similarity_api/src/app.py

```diff
@@ -1,5 +1,9 @@
+import logging
+
 from application import create_app
 
+logging.basicConfig(level=logging.INFO)
```

Comment:
Can logging level be set by an ENV and default to this value

Reply:
That way we can turn logging up or down without redeploy

Reply:
👍 b2726d0509c32c8284d625630376cc81f37bbeb8